app/services/active_storage/service/sdr_service.rb

```diff
@@ -15,6 +15,11 @@ def download(key, &block)
         end
       end
 
+      def delete(key)
+        Honeybadger.context(key)
```

Comment:
Should we use `#split_key` below to make it easier to figure out what we're seeing in the HB context? Or maybe the `key` is good enough?